riva/clients/tts/riva_tts_client.cc

```diff
@@ -47,9 +50,51 @@ DEFINE_string(
     zero_shot_audio_prompt, "",
     "Input audio file for Zero Shot Model. Audio length between 0-3 seconds.");
 DEFINE_int32(zero_shot_quality, 20, "Required quality of output audio, ranges between 1-40.");
+DEFINE_string(user_dictionary, "", " User dictionary containing graph-to-phone custom words");
 
 static const std::string LC_enUS = "en-US";
 
+std::string 
+ReadUserDictionaryFile(const std::string& dictionary_file)
+{
+  std::string dictionary_string;
+  if (!dictionary_file.empty()) {
+    std::ifstream infile(dictionary_file);
+
+    if (infile.is_open()) {
+      std::string line;
+      bool is_first_line = true;
+
+      while (std::getline(infile, line)) {
+        if (is_first_line) {
+          // Skip the first line it may contains headers
+          is_first_line = false;
+          continue;
+        }
+
+        std::istringstream iss(line);
+        std::string key, value;
+
+        if (std::getline(iss, key, ' ') && std::getline(iss, value)) {
+          // Trim leading and trailing whitespace from key and value
+          boost::trim(key);
+          boost::trim(value);
```

Comment:
we shouldn't need to use boost to trim leading/trailing whitespaces and can utilize some string library functions for this.

boost is available in container, but it is not linked while compiling clients standalone using this repo and I think that's what the build/test workflow is failing on this PR

Reply:
need to remove boost dependency to get the build workflow passing

Reply:
Removed as suggested

Reply:
need to remove from riva_tts_perf_client also and see if trim using STL lib functions can be done

Reply:
Can be done. For now i've removed it from perf clients